OpenSourceViewController+UISwift.swift

```diff
@@ -0,0 +1,34 @@
+//
```

Comment:
Can you please put this file into the source folder ? 
Otherwise I don't think it work with all dependencies tools.